cypress/integration/index.test.js

```diff
@@ -65,4 +65,32 @@ describe('percySnapshot', () => {
       '[percy] Error: 500 Internal Server Error'
     ]);
   });
+
+  describe('in interactive mode', () => {
+    beforeEach(() => {
+      Cypress.config('isInteractive', true);
```

Comment:
Rather than wholesale set this to true/false, we should capture the original value before this line and restore it in the after hook much like any other mock or stub would do. Especially since we're mocking an option that changes depending on how these tests are run.l